readme.md

```diff
@@ -15,8 +15,11 @@ source.
 
 ## Getting Started
 
-Your grinder build file should reside at `tool/grind.dart`, and may look
-something like:
+Your grinder build file should reside at `tool/grind.dart`. You can use grinder
+to create a simple, starting build script. Run `pub run grinder:init`; this will
```

Comment:
nit: might be worth putting `pub run grinder:init` on its own line, just so it stands out.